### PR TITLE
ENG-508: save secrets in DB by default

### PR DESCRIPTION
**Pull request by daabr**

### CODE REVIEW

File: internal/backend/secrets/svc.go [hunk 1]
```diff
@@ -26,18 +27,19 @@ var Configs = configset.Set[Config]{
...
 	case "aws":
 		impl, err = NewAWSSecrets(l, cfg)
-	// TODO(ENG-508): case "db"
+	case "file":
+		impl, err = NewFileSecrets(l, xdg.DataHomeDir())
 	case "vault":
 		impl, err = NewVaultSecrets(l, cfg)
 	default:
-		impl, err = NewFileSecrets(l, xdg.DataHomeDir())
+		impl, err = NewDatabaseSecrets(l, db)
```
Comment (itayd) [inline internal/backend/secrets/svc.go]: this should be when cfg.Type is empty or some specific string, not by default.

**Review by itayd — COMMENTED** [review]
are you sure you pushed everything?

File: internal/backend/secrets/svc.go (hunk 1 above)
Comment (daabr) [reply internal/backend/secrets/svc.go]: done

**Comment by daabr** [comment]
> are you sure you pushed everything?

Yes, the DB part was already done in #71.

**Review by itayd — APPROVED** [review]
lgtm, but would love to see a system test

**Pull request merged**